go.mod

```diff
@@ -1,13 +1,13 @@
-module github.com/HewlettPackard/hpegl-metal-terraform-resources
+module github.com/hewlettpackard/hpegl-metal-terraform-resources
 
 go 1.17
 
 require (
 	github.com/antihax/optional v1.0.0
 	github.com/hashicorp/terraform-plugin-docs v0.8.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.14.0
+	github.com/hewlettpackard/hpegl-metal-client v1.3.20-0.20220511202820-3b2ca9ca037a
```

Comment:
Did we want an untagged release of this? This version syntax usually means it is a commit after the v1.3.20 release.

Reply:
@neilgierman I will update this sometime tomorrow to reference the tagged version only.  Just trying to remove all old references and tests, temporary only.